helm-charts/dmr/templates/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+{{- $secrets := .Values.secrets | default dict -}}
+{{- $dmr := $secrets.dmr | default dict -}}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "dmr.fullname" . }}-appsettings-secrets
+  labels:
+    {{- include "dmr.labels" . | nindent 4 }}
+type: Opaque
+data:
+  # testKey: {{ default "" $dmr.testKey | b64enc | quote }}
```

Comment:
Is this line still required?
```suggestion
```

Reply:
Good spot, I've documented this this in the values file, so I will remove it